<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Studify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QComboBox" name="SearchBox">
      <property name="minimumHeight">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="StudentsList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="selectAllButton">
        <property name="text">
         <string>ВЫБРАТЬ ВСЕ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generatePdfButton">
       <property name="icon">
        <iconset>
            <normaloff>:/pdf.ico</normaloff>
        </iconset>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
